branches.go

```diff
@@ -36,6 +37,36 @@ func newBranch(c cliContext, api branchAPI) error {
 	return nil
 }
 
+func mergeBranch(c cliContext, api branchAPI) error {
+	name := c.Args().First()
+	name = strings.TrimSpace(name)
+
+	if name == "" {
+		return cli.NewExitError("Branch name cannot be blank", 1)
+	}
+
+	branches, err := api.GetBranches(name)
+
+	if err != nil {
```

Comment:
Same question about preventing attempting to merge `main` locally

Reply:
Same points as for handling deleting main. Added to the [ticket](https://rainforest.atlassian.net/browse/RF-25651).